Umee-Bots/Deposit-Borrow-Flashloan/README.md

```diff
@@ -0,0 +1,44 @@
+# Deposit/Borrow In The Same Transaction as Flash Loan
+
+## Description
+
+This bot monitors deposit or borrow events in the same transaction along with a flash loan event.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- UMEE-14-1
+
+  - Fired when `Deposit` and `FlashLoan` events are emitted in the same transaction
+  - Severity is always set to "info"
+  - Type is always set to "info"
+  - metadata:
+    - realizedEvents: Detected event pair.
+    - from: Address of the sender of the transaction.
+    - to: Address of the message call’s recipient.
+
+- UMEE-14-2
+  - Fired when `Borrow` and `FlashLoan` events are emitted in the same transaction
+  - Severity is always set to "info"
+  - Type is always set to "info"
+  - metadata:
+    - realizedEvents: Detected event pair.
```

Comment:
Related to a comment below, just a note to remember to update the README if this is changed.